rules/linux/lnx_install_root_certificate.yml

```diff
@@ -0,0 +1,27 @@
+title: Install Root Certificate
+id: 78a80655-a51e-4669-bc6b-e9d206a462ee
+description: Detects installed new certificate
+author: Ömer Günal, oscd.community
+date: 2020/10/05
+tags:
+    - attack.defense_evasion
+    - attack.t1553.004
+level: low
+logsource:
+    product: linux
+detection:
+    selection:
+        - CommandLine|contains|all:
+          - 'mv '
+          - '/usr/local/share/ca-certificates'
```

Comment:
this section could be completely removed, as there are many ways to move/create file in this folder. 
the key command here is what you have in `selection2`

Reply:
Thanks!